lib/bambora/v1/profile_resource.rb

```diff
@@ -54,8 +54,8 @@ def initialize(client:, api_key:)
       # @see https://dev.na.bambora.com/docs/guides/payment_profiles
       #
       # @return [Hash] Indicating success or failure of the operation.
-      def create(card_data)
-        client.post(path: sub_path, body: card_data, api_key: api_key)
+      def create(payment_profile_data)
```

Comment:
Updating this because this is also used to create a `bank` payment profile. 

This doesn't use a keyword argument so we don't need to update any code that uses this method aka the HiMama app.

Reply:
Nice clean-up!